manage.py

```diff
@@ -36,6 +37,27 @@ def add_event_identifier():
         save_to_db(event)
 
 
+@manager.option('-n', '--name', dest='name', default='all')
+@manager.option('-s', '--switch', dest='switch', default='off')
+def module(name, switch):
+    keys = [i.name for i in Module.__table__.columns][1:]
+    convey = {"on": True, "off": False}
+    if switch not in ['on', 'off']:
+        print("Choose either state On/Off")
+
+    elif name == 'all':
+        for key in keys:
+            setattr(Module.query.first(), key, convey[switch])
+            db.session.commit()
```

Comment:
Move commit to last line instead of commiting in each iteration

Reply:
Implemented